rest-api/requirements.txt

```diff
@@ -0,0 +1,12 @@
+dataclasses==0.7
```

Comment:
Are fixed version really required? Especially for python-bareos, which will be shipped together with the rest-api?

Reply:
It is not required but basically a good idea to pip freeze dependency installed in your virtual environment to avoid trouble and headache. Some versions of packages will not work well with other packages and their respective versions.

Reply:
For the package which is pip installed, dependencies should not rely on the requirements.txtbe and instead be declared like described in https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html?highlight=declaring%20dependencies#declaring-required-dependency .

Reply:
requirements.txt adapted, only starlette needs a max version number. Packaging of rest-api for pip will be next step on the list.